### DISCUSSION TITLE
how to add index

### DISCUSSION THREAD

**Discussion by llllly26** [facebookresearch/faiss#1946] [Q&A]
hi,I want to know when I have an index file,and I have a batch of new data and want to add it to the index,how to do? use `index.add()` directly? if i need to train? and I want to know the principle of `index.train`,thank you.

**Comment by kingafy** [comment]
Even I have a similar requirement. How to add new delta file to existing index instead of reprocessing the entire file.

**Comment by PankajJ08** [comment]
Any update on the issue?
